### ISSUE TITLE
Scrolling and input on touch screens

### ISSUE THREAD

**Issue by pjrobertson** [audreyt/ethercalc#196]
The scroll bars do not seem to work on touch screens. Also, tapping a cell doesn't bring up the keyboard. Perhaps a double tap should bring up the keyboard (so that a single tap can still be used to select a cell)

Also, the CSS probably needs improving a little so that it looks better on small screens.

**Comment by audreyt** [comment]
Yes, this is a long-standing issue. Tapping the input box can bring up input, but scrolling (via dragging the handle) is far from intuitive.

CSS we can adjust first so that at least the reading experience is better.
